Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2585 @@
+             result.
+     """
+
+    sha2 = args.get('file', '')
+    if get_hash_type(sha2) != 'sha256':
+        raise ValueError(f'SHA256 value:{sha2} is invalid')
+
+    payload = {
+        'action': 'analyze',
+        'targets': argToList(sha2)
+    }
+    response = client.submit_file_to_sandbox_analyze(payload)
+
+    # Get Issue Sandbox Command
+    title = "Issue Sandbox Command"
+    summary_data = {
+        'sha2': sha2,
+        'command_id': response.get('command_id'),
+        'command_type': 'Issue Sandbox Command'
+    }
+    headers = list(summary_data.keys())
+    column_order = [camelize_string(column) for column in headers]
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.SandboxIssue',
+        outputs_key_field='command_id',
+        outputs=summary_data,
+        readable_output=tableToMarkdown(title, camelize(summary_data, '_'), headers=column_order, removeNull=True),
+        raw_response=response
+    )
+
+
+# ScheduledCommand
+def run_polling_command(client: Client, args: dict, cmd: str, status_func: Callable, results_func: Callable):
+    """
+    This function can handle the polling flow.
+    After the first run, progress will be shown through the status command.
+    The run_polling_command function check the file scan status and will run until status is not 'Completed'.
+    It returns a ScheduledCommand object that schedules the next 'results' function until the polling is complete.
+    Args:
+        client: Symantec EDR client object
+        args: the arguments required to the command being called
+        cmd: the command to schedule by after the current command
+        status_func : The function that check the file scan status and return either completed or error status
+        results_func: the function that retrieves the verdict based on file sandbox status
+
+    Returns:
+        return CommandResults
+    """
+    demisto.debug(f'-- Polling Command --\nArguments : {args}')
+    ScheduledCommand.raise_error_if_not_supported()
+    interval_in_secs: int = int(args.get('interval_in_seconds', DEFAULT_INTERVAL))
+    timeout_in_seconds: int = int(args.get('timeout_in_seconds', DEFAULT_TIMEOUT))
+
+    # Check for ongoing file scanning command_id if exist
+    if pre_cmd_id := demisto.getIntegrationContext().get('command_id'):
+        args['command_id'] = pre_cmd_id
+
+    # first run ...
+    if 'command_id' not in args:
+        outputs: Any[object] = issue_sandbox_command(client, args).outputs
+        command_id = outputs.get('command_id')
+
+        if command_id is not None:
+            if global_integration_context := demisto.getIntegrationContext():
+                global_integration_context['command_id'] = command_id
+                demisto.setIntegrationContext(global_integration_context)
+            else:
+                demisto.setIntegrationContext({'command_id': command_id})
```

Comment:
does this overwrite the whole integration context? or just add?

Reply:
If the global integration context data exists, then not to overwrite.